torchbox/torchbox.com
# Tidy up unused templates and add very basic styling for the password required page

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1426776476)

Note this ticket is not in sprint but it turned out you could navigate to a page in the pattern library that threw an error because this was not done.

### Description of Changes Made

- Remove any unused templates left over from the old build
- Get rid of the blog-item template that is an unnecessary extra include
- Do some very basic styling for the forgotten password page just because it was quick and it's another one to cross off.

### How to Test
Grep the codebase for any templates that are removed and ensure they are not referenced anywhere
Check the password requred styling: http://localhost:8000/pattern-library/pattern/patterns/pages/wagtail/password_required.html

Note that in a screen reader the password required form does not focus on the error message on submit. However, this would require extra JS which would be a future enhancement.

### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot 2024-03-25 at 12 39 40](https://github.com/torchbox/torchbox.com/assets/771869/938c6dc4-f817-42ea-b0da-3dcf0011bf80)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
